utils/automatic-release/test/process.test.js

```diff
@@ -108,8 +108,6 @@ jest.mock('../lib/versionAddCommitTagPackage', () =>
   jest.fn(() => Promise.resolve())
 );
 
-const versionAddCommitTagPackage = require('../lib/publishPackage');
```

Comment:
We don't want to let jest mock this file anymore? why?

Reply:
It's still mocked in the line above; however, versionAddCommitTagPackage is unused as a variable.  This was caught via linting.

Reply:
There's certainly more clean-up we can do with the mocks though.  They are all globally scoped, which caused some problems, but would take more time to fix than what I did in this PR.